pkg/instance/instance_test.go

```diff
@@ -30,10 +30,11 @@ func TestFuzzerCmd(t *testing.T) {
 	flagExecutor := flags.String("executor", "./syz-executor", "path to executor binary")
 	flagSignal := flags.Bool("cover", false, "collect feedback signals (coverage)")
 	flagSandbox := flags.String("sandbox", "none", "sandbox for fuzzing (none/setuid/namespace)")
+	flagSandboxArg := flags.Int("sandbox_arg", 0, "")
 	flagDebug := flags.Bool("debug", false, "debug output from executor")
 	flagV := flags.Int("v", 0, "verbosity")
 	cmdLine := OldFuzzerCmd(os.Args[0], "/myexecutor", "myname", targets.Linux, targets.I386, "localhost:1234",
-		"namespace", 3, true, true, false, 5)
+		"namespace", 23, 3, true, true, false, 5)
```

Comment:
Hard to understand "23" here. Lets use some named constant.

Reply:
'sandbox_arg' doesn't have a list of predefined values. It's a generic argument name, which is present in many programming languages and frameworks. Some call it 'tag', others 'arg1', 'e' or lpThreadParameter (CreateThread in Win32 kernel).

It is up to executor implementation to decide what it means. As of now Android sandbox uses two values: '1' - system account settings and everything else - a restricted app. Most likely soon we introduce more values. I do not think we should restrict a range of values, because in some cases it could be a token value passed as-is to a function inside executor and it could be negative. Not 100% sure, however, whether such case is possible in Android or in any of the supported OSes, but suspect it is viable. 'sandbox_arg' can be trusted since it is not provided by a malicious agent from the Internet. 

As of now 'sandbox_arg' is limited to Android only, but it has potential to be used for other OSes if users of syzkaller want to fuzz only subset of OS functionality.

23 carries no particular meaning. This test case validates that a two digit integer can be passed. I will add a comment.